### PR TITLE
Update Prerequisites section in README.md

### PR DESCRIPTION
**Pull request by maheshschand**
This resolves issue partially in #40.

### CODE REVIEW

**Comment by maheshschand** [comment]
@sagar-sehgal Could you please merge this issue?

**Comment by rajatkb** [comment]
@sagar-sehgal  kindly review and give a GO on this one.

File: README.md [hunk 1]
```diff
@@ -8,19 +8,40 @@ Conference-Notify will be an open source web based application that will aggrega
...
+#### Atlas
+* MongoDB Atlas is the global cloud database service for modern applications.
+1. Go to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)
+2. Create an account or sign in into your mongoDB account.
+3. Depending on the usage, select various features.  
+For more information, Visit [MongoDB Atlas Docs](https://docs.atlas.mongodb.com/)
+
+#### Python 3.6
+1. Download [Python](https://www.python.org/downloads/) 
+2. Install it.
```
[the remark this answers is not in the document]
Comment (maheshschand) [reply README.md]: But in windows, the path is automatically set by the setup file. I will update this to ubuntu!

**Comment by maheshschand** [comment]
@sagar-sehgal I made necessary changes. Please look into it.

**Comment by sagar-sehgal** [comment]
@maheshschand Why the PR was closed? The new changes in [this PR](https://github.com/rajatkb/Conference-Notify/pull/44/commits/8fbd9b580d80134002c5eccc428f9a5c12245b6e?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) seem to be fine.

**Comment by maheshschand** [comment]
I well open a new one.

**Pull request closed without merging**